### PR TITLE
Feature/surplus

### PR DESCRIPTION
**Pull request by ChiaraDipi**

### CODE REVIEW

File: src/app/services/xml-parsers/basic-parsers.ts
Comment (szenzaro) [inline src/app/services/xml-parsers/basic-parsers.ts:133]: Shouldn't this be inside "editorial-parsers"?

File: src/app/components/surplus/surplus.component.html
Comment (giuliac89) [reply src/app/components/surplus/surplus.component.html:3]: I agree!

**Review by szenzaro — APPROVED** [review]

**Comment by ChiaraDipi** [comment]
Since surplusComponent can be subject to custom editorial conventions, I would rewrite the whole component after merging of PR #93. @szenzaro since you already approved the PR, if you prefer I can close this one and open a new one once I've done the changes.

**Pull request merged**
